### PR TITLE
Create base registration app from cookiecutter

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: familyconnect_registration/urls.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import os
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+admin.site.site_header = os.environ.get('REGISTRATIONS_TITLE',
+                                        'familyconnect_registration Admin')
```
Comment (imsickofmaps) [inline familyconnect_registration/urls.py]: Neaten to `FamilyConnect Registration Admin` please

File: registrations/tests.py [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    def setUp(self):
+        self.adminclient = APIClient()
+        self.normalclient = APIClient()
+        self.otherclient = APIClient()
+
+
+class AuthenticatedAPITestCase(APITestCase):
+
+    def make_source_adminuser(self):
+        data = {
```
Comment (imsickofmaps) [inline registrations/tests.py:23]: make these a bit more terse as discussed please

File: registrations/tests.py [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+    def test_get_source_adminuser(self):
+        # Setup
+        source = self.make_source_adminuser()
+        # Execute
+        response = self.adminclient.get('/api/v1/source/%s/' % source.id,
+                                        format='json',
+                                        content_type='application/json')
+        # Check
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (imsickofmaps) [inline registrations/tests.py:161]: Please can we check some of the `response.data["field"]` values are correctly serialized

**Comment by imsickofmaps** [comment]
Minor changes then :+1: when Travis happy

**Pull request merged**